charts/operators-installer/templates/Job_installplan-approver.yml

```diff
@@ -0,0 +1,90 @@
+{{- range .Values.operators }}
+{{- if eq .installPlanApproval "Manual" }}
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: installplan-approver--{{ .name }}
+  annotations:
+    "helm.sh/hook": post-install,post-upgrade
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+    "helm.sh/hook-weight": "0"
+spec:
+  completions: 1
+  backoffLimit: 5
+  parallelism: 1
+  activeDeadlineSeconds: 30
+  template:
+    spec:
+      containers:
+      - name: installplan-approver
+        image: registry.redhat.io/openshift4/ose-cli:v4.10
```

Comment:
Let's go ahead an parameterize the image field

Reply:
done.